pipelines/jenkins/JenkinsAnchoreCICD.txt

```diff
@@ -0,0 +1,98 @@
+pipeline {
+        agent any
+        environment {
+            //Define some reusable variables here 
+            IMAGENAME="kaizencont/goutil"
+            WAIT_TIMEOUT=10
+            ANCHORE_URL="http://10.0.0.15:8080/v1/"
+            dockerImage = '' 
+            registryCredential = 'dockerhub'
+        }
+        stages {
+             stage ('Build') {
+                steps {
+                    println ("Building image for configured project...")
+                    
+                    // Performs a checkout of the project from the specific git repository
+                    git 'https://github.com/jeremybryan/go-aws-utils.git/'
+                
+                    // Perform a docker build of the project assumes the Dockerfile is located in the project's root directory
+                    script { 
+                            dockerImage = docker.build IMAGENAME+":"+ TAG
+                    }
+                
+                    // Push the created image to the registry configured with registryCredential
+                    script {
+                        docker.withRegistry( '', registryCredential ) { 
+                                dockerImage.push() 
+                        }
+                    }
+                }
+            }
+            
+            stage("Verifying Anchore Availability") {
```

Comment:
@jeremybryan do you think we should add a comment here that this isn't necessary or to modify this if people want to block on service unavailability? Just out of an abundance of caution since it's an example

Reply:
@found-it This was really just for demonstration purposes … it’s not strictly necessary more just to show how one could do that, I’ll add some additional comments throughout to indicate this fact.

Reply:
Sounds good!